src/components/Dropdown.js

```diff
@@ -41,12 +41,12 @@ export default function Dropdown({ onClick = () => {}, disabled = false }) {
                       }}
                     >
                       <a className="flex cursor-pointer justify-between w-full px-4 py-2 text-sm leading-5 text-left">
-                        Export MP3
+                        Download MP3
                       </a>
                     </Menu.Item>
                     <Menu.Item>
                       <a className="flex cursor-pointer justify-between w-full px-4 py-2 text-sm leading-5 text-left">
-                        Export OGG
+                        Download WAV
```

Comment:
I think I rebased things incorrectly, since we're seeing commits originally not in this branch.

Reply:
yea our commit history would be harder to read but it shouldn't break anything. rebasing is tricky!

Reply:
this is fine tho, don't bother fixing this

Reply:
I think I fixed it with `git reset`, then `git rebase master`, and then `git push -f`. I checked to make sure `git log` returned a history that made sense, at least to me, before doing this.

It seems to have worked, since this pull request is now only showing commits from itself.

I had the option to do `git push -f` earlier, but I tried rebasing use `git pull` this time to get exposure with it, since I've used `git push -f` to fix potential merge conflicts before, and since I wanted to avoid use of the `-f` flag, but it ended up with a messier history.